ci/scripts/PKGBUILD

```diff
@@ -73,26 +73,11 @@ build() {
   # set the appropriate compiler definition.
   export CPPFLAGS="-DUTF8PROC_STATIC"
 
-  # This is the difference between rtools-packages and rtools-backports
-  # Remove this when submitting to rtools-packages
-  if [ "$RTOOLS_VERSION" = "35" ]; then
-    export CC="/C/Rtools${MINGW_PREFIX/mingw/mingw_}/bin/gcc"
-    export CXX="/C/Rtools${MINGW_PREFIX/mingw/mingw_}/bin/g++"
-    export PATH="/C/Rtools${MINGW_PREFIX/mingw/mingw_}/bin:$PATH"
-    export CPPFLAGS="${CPPFLAGS} -I${MINGW_PREFIX}/include"
-    export LIBS="-L${MINGW_PREFIX}/libs"
-    export ARROW_GCS=OFF
-    export ARROW_S3=OFF
-    export ARROW_WITH_RE2=OFF
-    # Without this, some dataset functionality segfaults
-    export CMAKE_UNITY_BUILD=ON
-  else
-    export ARROW_GCS=ON
-    export ARROW_S3=ON
-    export ARROW_WITH_RE2=ON
-    # Without this, some compute functionality segfaults in tests
-    export CMAKE_UNITY_BUILD=OFF
-  fi
+  export ARROW_GCS=ON
```

Comment:
These don't need to be defined out here anymore, they can just be set in the cmake below like the other flags. If you want them configurable, you can set them like `-DARROW_GCS="${ARROW_GCS:-ON}" like we do in inst/build_arrow_static.sh.